### DISCUSSION TITLE
Thank You.

### DISCUSSION THREAD

**Discussion by retropwned** [Open-Shell/Open-Shell-Menu#1012] [General]
I just wanted to take a moment to say thank you, to all of you, for keeping classic shell alive. i know it takes hard work and dedication but I cannot think of any other program that is more important to the power user in the windows environment. HL3 Lives.

**Comment by blackcrack** [comment]
if you know anyone where be in able to programming here, speak with him.
i guess well programmed patches be welcome.
best
